Mathlib/Init/Data/Int/Basic.lean

```diff
@@ -11,18 +11,13 @@ import Std.Data.Int.Order
 
 open Nat
 
--- TODO: backport?
 #align int.neg_succ_of_nat Int.negSucc
 
--- @[inherit_doc]
+@[inherit_doc]
 notation "ℤ" => Int
```

Comment:
I don't think this is right; it should use `extend_doc` or something, so that we can add a remark that mentions the notation in the docstring.